src/tdastro/base_models.py

```diff
@@ -95,30 +78,49 @@ def add_parameter(self, name, value=None, required=False, **kwargs):
             value = kwargs[name]
         if value is not None:
             if isinstance(value, types.FunctionType):
+                # Case #1: If we are getting from a static function, sample it.
```

Comment:
```suggestion
                # Case 1: If we are getting from a static function, sample it.
```
Tiniest tiniest nit, but while it's on the mind